src/app/pages/loading-waiting-states/skeleton-loader/skeleton-loader.component.html

```diff
@@ -101,7 +101,7 @@ <h2 class="example-title">List Items</h2>
                 </mat-nav-list>
             </mat-card>
             <h2 class="example-title"></h2>
-            <mat-card class="list-item-example-two">
+            <mat-card class="list-item-example-two" [hidden]="!showPlaceHolder">
                 <mat-nav-list [style.paddingTop.px]="0">
                     <pxb-info-list-item
                         divider="full"
```

Comment:
Line 157 of this file, there should be no divider in an info list item when it's at the end of the list